content/spin/v2/python-components.md

```diff
@@ -28,23 +27,6 @@ With <a href="https://www.python.org/" target="_blank">Python</a> being a very p
 
 [**Want to go straight to the Spin SDK reference documentation?**  Find it here.](https://fermyon.github.io/spin-python-sdk/)
 
-**Please note:** There is a blog article [introducing the release of this Spin Python SDK](https://www.fermyon.com/blog/spin-python-sdk) if you are interested in some further reading; in addition to this technical documentation.
-
-## Spin's Python Plugin
```

Comment:
You probably want to replace this with a mention of `componentize-py` rather than saying nothing...!

Reply:
Good idea, I will do that.